Tests/AutoTests/AccessebilityTest/accessiblity-RadioNavigationalWarnings.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test} from '@playwright/test';
+import * as app from "../../Configuration/appConfig.json";
+import { checkA11y, injectAxe, Options } from 'axe-playwright';
+import RadioNavigationalWarnings from '../../pageObject/RadioNavigationalWarnings.page';
+
+test.describe("A11y tests", ()=> {
+  const defaultCheckA11yOptions: Options = {
+    axeOptions: {
+      runOnly: {
+        type: 'tag',
+        values: ['wcag2aa'],
+      },
+      reporter: 'v2'
+    },
+    detailedReport: true,
+    detailedReportOptions: { html: true }
+  };
+ 
+  test.beforeEach(async ({page}) => {
+    await page.goto(app.url);    
+    console.log(page.url());
```

Comment:
Could you please remove the console.log()

Reply:
resolved